docs/concepts/crds/backupsession.md

```diff
@@ -0,0 +1,139 @@
+
+You can also create a `BackupSession` object manually to trigger backup instantly.
+
+>In this tutorial, we are going to use **host** word to indicate an entity (pod) whose data is backed up.
+
+>- For `Deployment`, `ReplicationController` and `ReplicaSet`, backup process run in only one pod. This pod is referred as **host-0**.
+>- For `StatefulSet`, backup process run in all pods. In this case, **pod-0** is known as **host-0**, **pod-1** is known as **host-1**, **pod-2** is known as **host-2** and so on.
+>- For `DaemonSet`, backup process run in all daemon pods. In this case, the **node name** where the pod is running act as their **host** name.
+>- For database or stand-alone PVC, backup is done by a job. In this case, the backup job's pod is known as **host-0**.
+
+## BackupSession CRD Specification
+
+Like any official Kubernetes resource, a `BackupSession` has `TypeMeta`, `ObjectMeta` and `Spec` , `Status` sections.
+
+A sample `BackupSession` created for backing up a Deployment's data is shown below,
+
+```yaml
+apiVersion: stash.appscode.com/v1beta1
+kind: BackupSession
+metadata:
+  name: deployment-backup-2do7af
+  namespace: demo
+  labels:
+    backup-configuration: deployment-backup
+spec:
+  backupConfiguration:
+    name: deployment-backup
+status:
+  phase: Succeeded
+  totalHosts: 1
+  sessionDuration: 3.861369766s
+  stats:
+  - hostname: host-0
+    phase: Succeeded
+    duration: 3.861369766s
+    snapshots:
+    - directory: /source/data
+      fileStats:
+        modifiedFiles: 0
+        newFiles: 0
+        totalFiles: 2
+        unmodifiedFiles: 2
+      name: ea5381cc
+      processingTime: 0m0s
+      size: 30 B
+      uploaded: 0 B
+```
+
+Here, we are going to describe some important sections of `BackupSession` crd.
+
+### BackupSession `Metadata`
+
+#### metadata.name
+
+`metadata.name` indicates the name of the `BackupSession`. This name is automatically generated by respective `CronJob` and it follows the following pattern: `<BackupConfiguration name>-<random suffix>`.
+
+#### metadata.namespace
+
+`metadata.namespace` indicates the name of the `BackupSession`. It is same as the namespace of respective `BackupConfiguration` object.
+
+#### metadata.labels
+
+`metadata.labels` holds respective `BackupConfiguration` name as label. Stash backup sidecar container use this label to watch only the BackupSessions of that `BackupConfiguration`.
+
+>If you create `BackupSession` manually to trigger a backup instantly, make sure that you have added `backup-configuration: <BackupConfiguration name>` label to your `BackupSession`. Otherwise, it will not trigger backup for workloads (those resources that are backed up using sidecar).
```

Comment:
I think this should have a `stash.appscode.com/` prefix. Let's review all the annotations and labels used.